src/seeds/pipeline/version.txt

```diff
@@ -1 +1 @@
-0.14.0
\ No newline at end of file
+0.14.1
```

Comment:
This should probably be a minor release, right? Also, the "project" seed should also be updated to have the up-to-date version of the pipeline library.

Reply:
Pipeline seed now updated to `0.15.0`